linchpin/api/__init__.py

```diff
@@ -412,6 +414,53 @@ def lp_down(self, pinfile, targets='all'):
         pass
 
 
+    def lp_fetch(self, src, fetch_type, root):
+        if root is not None:
+            root = list(filter(None, root.split(',')))
+
+        dest = self.ctx.workspace
+        if not os.path.exists(dest):
+            raise LinchpinError(dest + " does not exist")
+
+        fetch_aliases = {
+            "topology": self.get_evar("topologies_folder"),
+            "layout": self.get_evar("layouts_folder"),
+            "resources": self.get_evar("resources_folder"),
+            "hooks": self.get_evar("hooks_folder"),
+            "workspace": "workspace"
```

Comment:
I noticed this in the demonstration. Mostly a nitpick, but we use plural values for all things here, unless there's a good reason not to do so. In this case, inventories and layouts should be the thing we do with fetch. Additionally, I'd like to comment this to be a sub-command in [Click](http://click.pocoo.org) in the future.  Can you add a comment here to eventually move this into a sub-command?